lib/form.js

```diff
@@ -160,12 +160,25 @@ _.extend(Form.prototype, {
     saveValues: function (req, res, callback) {
         callback();
     },
-    getNextStep: function (req) {
+    getNextStep: function (req, res) {
         var next = this.options.next || req.path;
         if (req.baseUrl !== '/') {
             next = req.baseUrl + next;
         }
-        return next;
+        var forks = this.options.forks || [];
+
+        var evalCondition = function evalCondition(condition) {
```

Comment:
just being pedantic, could this just be a named function declaration rather than a named literal?